### PR TITLE
few improvements to active users

### PR DESCRIPTION
**Pull request by rawxfish**
A few changes to make active users backwards compatible or more robust.

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,5 +1,4 @@
 release: python manage.py migrate --noinput
 web: gunicorn cardboard.wsgi --log-file -
-worker: celery -A cardboard worker -l INFO --without-heartbeat --without-gossip --without-mingle --concurrency 2
```
Comment (erwa) [inline Procfile:3]: Nice, glad you were able to combine the workers

File: scripts/add_service_account_contacts.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import datetime
+
+import googleapiclient
+from django.conf import settings
+from google.oauth2 import service_account
+
+from accounts.models import Puzzler
+
+# For adding all registered user into service account's contact list
```
Comment (erwa) [inline scripts/add_service_account_contacts.py:9]: user -> users

File: google_api_lib/tasks.py [hunk 3]
```diff
@@ -461,21 +462,48 @@ def extract_id_from_drive_item_name(drive_item_name) -> Optional[str]:
...
     UserModel = get_user_model()
     try:
         return UserModel.objects.get(social_auth__uid=uid)
     except UserModel.DoesNotExist:
+        pass
+
+    # Fall back to People API
+    # Note that this API depends on the email being added as service account's contact list
+    # It also has a tight request limit
+    time.sleep(2)
```
Comment (erwa) [inline google_api_lib/tasks.py:475]: Why do we need to sleep here?

Comment (rawxfish) [reply google_api_lib/tasks.py:475]: So this branch of the code should only run for backward compatibility with previous year's data. When that happens, we actually call the People API in quick succession and it's easy to hit the API's 90 request / minute quota, probably could do with just 1 second though.

**Review by asdfryan — APPROVED** [review]

File: google_api_lib/tasks.py (hunk 3 above)
Comment (erwa) [reply google_api_lib/tasks.py]: Oh gotcha. Could you mention that the sleep is to prevent exceeding Google API limits?

Comment (erwa) [reply google_api_lib/tasks.py]: I guess you mentioned the tight request limit, nvm

**Pull request merged**